airbyte-scheduler/app/build.gradle

```diff
@@ -28,11 +28,17 @@ rootProject.file('.env.dev').withInputStream { env.load(it) }
 
 run {
     // default for running on local machine.
-    environment "CONFIG_ROOT", "${env.DEV_ROOT}/data"
-    environment "WORKSPACE_ROOT", "${env.DEV_ROOT}/workspace"
-    environment "WORKSPACE_DOCKER_MOUNT", env.WORKSPACE_DOCKER_MOUNT
-    environment "VERSION", env.VERSION
     environment "DATABASE_USER", env.DATABASE_USER
     environment "DATABASE_PASSWORD", env.DATABASE_PASSWORD
-    environment "DATABASE_URL", "jdbc:postgresql://localhost:5432/airbyte"
+    // we map the docker pg db to port 5433 so it does not conflict with other pg instances.
+    environment "DATABASE_URL", "jdbc:postgresql://localhost:5433/${env.DATABASE_DB}"
+    environment "WORKSPACE_ROOT", env.WORKSPACE_ROOT
+    environment "WORKSPACE_DOCKER_MOUNT", env.WORKSPACE_DOCKER_MOUNT
+    environment "LOCAL_DOCKER_MOUNT", env.LOCAL_DOCKER_MOUNT
+    environment "CONFIG_ROOT", "/tmp/airbyte_config"
```

Comment:
for my own knowledge - are the `airbyte-scheduler` and `airbyte-server` config roots identical because we share information via the volume?